app/views/questions/_comment.html.erb

```diff
@@ -26,7 +26,10 @@
 
   </div>
   <div class="inline pull-xs-right" style="vertical-align: top">
-    <p><% if answer_id == 0 && current_user && (current_user.role == "admin" || current_user.role == "moderator" || comment.uid == current_user.uid)%>
+	<a rel="tooltip" title="Flag as spam" class="btn btn-sm btn-default btn-flag-spam-<%= comment.id %>" href="mailto:moderators@publiclab.org?subject=Reporting+spam+on+Public+Lab&body=Hi,+I+found+this+comment+that+looks+like+spam+or+needs+to+be+moderated:+https://publiclab.org/<%= comment.parent.path %>#c<%= comment.cid %>+by+https://publiclab.org/profile/<%= comment.author.username %>+Thanks!">
+		<i class="fa fa-flag"></i>
+	</a>
+	<p><% if answer_id == 0 && current_user && (current_user.role == "admin" || current_user.role == "moderator" || comment.uid == current_user.uid)%>
```

Comment:
Do you think we can do some changes to make view cleaner by aligning all three buttons in a line? Actually, the delete button is visible to you because you are currently logged in as 'moderator' from localhost. And, I think as moderator already has the power to delete the comment, so flag option is not needed for moderator and admin. So, here's what you can do -

1. Add if statement so that button will appear only if the current user is not admin or moderator. The if statement would look something like this- `!current_user || current_user.role != "admin" && current_user.role != "moderator"` 

2. Paste the screenshot logging in as user and moderator, and view without logging in.

Thanks!